xrpl/openapi-codegen/models/memo.py

```diff
@@ -0,0 +1,29 @@
+"""Model for Memo."""
+
+from dataclasses import dataclass
+from typing import Optional
+from xrpl.models.base_model import BaseModel
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class Memo(BaseModel):
+    memo_data: Optional[str] = None
```

Comment:
nit: if a type doesn't have a description, can there be a newline between the `class` line and the first param? Easier to read that way

Reply:
Agree. But all of them should have a description imo so probably will need to add to the spec